src/App.js

```diff
@@ -1,22 +1,27 @@
+import { useState } from 'react';
 import logo from './logo.svg';
 import './App.css';
 
 function App() {
+  const [message, setMessage] = useState('');
+
+  const handleClick = () => {
+    setMessage('Hello!!!');
+    setTimeout(() => setMessage(''), 1000);
+  };
+
   return (
     <div className="App">
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
+
+        {message}
+
         <p>
```

Comment:
Hello! I wouldn't usually put a button within a p tag, so might suggest that you get rid of the ```p``` tag and leave the button as it's own element.  

Alternatively, if you're looking to add more to it, a ```div``` might be more appropriate.